testcases/ipv6/test.yml

```diff
@@ -68,8 +68,28 @@
   with_items:
     - "{{ hostvars[inventory_hostname]['groups']['backend_servers_ipv6'] }}"
 
-# TODO: check the ipv6 member's status from neutron db is ONLINE after creation.
-#   reference: https://github.com/zongzw/f5-oslbaasv2-tools/blob/master/f5-oslbaasv2-batchops/scripts/ansible-scripts/if-exists-resource.sh
+- name: "get {{ pool }}-ipv6 's id"
```

Comment:
1. These tasks should be after `- name: "{{ testcasename }}: waiting for ipv6 resource created/updated"`
2. we should check `member_id == <id>` not (pool_id and then  `- "'ONLINE' in '{{ item }}'"`)
3. L82 is too long.
4. Is it possible to use `community.mysql.mysql_query`(https://docs.ansible.com/ansible/latest/collections/community/mysql/mysql_query_module.html) and assert it.